pages/wishes/index.vue

```diff
@@ -122,6 +122,15 @@ export default {
     },
     avatarColor(fullName) {
       return randomMC.getColor({ text: fullName })
+    },
+    wishDelete(wish){
```

Comment:
Check for accessToken at this point. In future, we should to check it at VueX actions.
Allow deleting of wish, if accessToken is not null, otherwise `this.throwAccessDenied()` and stop loading in delete button.

Example can be seen there.
```js
if (!this.accessToken) {
      this.loading = this.checkForToken()
    } else if (this.user) {
      this.getWishes().then(() => {
        this.loading = false
      }).catch(() => {
        this.loading = false
      })
    } else {
      this.getUser().then(() => {
        this.getWishes().then(() => {
          this.loading = false
        }).catch(() => {
          this.loading = false
        })
      }).catch(() => {
        this.loading = false
        this.throwAccessDenied()
      })
    }
```